lib/use-cases.typ

```diff
@@ -40,6 +40,19 @@
 	)
 }
 
+#let usecase_flows(basic: (), alt: ()) = {
+	//typst keeps complaining: "expected content, found array". Have no idea why this doesn't work or how to fix
+	figure(
+		table(
+			columns: 2,
+			[*Basic Flow*], [*Alternate Flow*],
+			[#list(basic)], [#list(alt)]
```

Comment:
`list` takes a `..content` for the items in the list, similar to how `table` does. So with literals you would call it like `list([foo], [bar], [baz])`. To call it with an array you have to spread it: `list(..basic)`